templates/microprofile-kumuluzee/files/README.md

```diff
@@ -1,14 +1,22 @@
-# KumuluzEE template for Platform.sh
+# KumuluzEE for Platform.sh
 
-This project provides a starter kit for KumuluzEE MicroProfile projects hosted on [Platform.sh](http://platform.sh).
+This project provides a starter kit for KumuluzEE Eclipse MicroProfile projects hosted on [Platform.sh](http://platform.sh).
 
-We include recommended .platform.app.yaml and .platform files that should suffice for most use cases. You are free to tweak them as needed for your particular site.
+KumuluzEE is a lightweight framework for developing microservices using standard Java, Java EE / Jakarta EE technologies and migrating existing Java applications to microservices. KumuluzEE packages microservices as standalone JARs. KumuluzEE microservices are lightweight and optimized for size and start-up time. 
 
-## Platform.sh configuration files
+## Services
+
+* Java 8
+
+## Customizations
+
+The following files and additions make the framework work.  If using this project as a reference for your own existing project, replicate the changes below to your project.
 
 * [`.platform/routes.yaml`](.platform/routes.yaml): Platform.sh allows you to define the [routes](https://docs.platform.sh/configuration/routes.html).
 * [`.platform/services.yaml`](.platform/services.yaml):  Platform.sh allows you to completely define and configure the topology and [services you want to use on your project](https://docs.platform.sh/configuration/services.html).
 * [`.platform.app.yaml`](.platform.app.yaml): You control your application and the way it will be built and deployed on Platform.sh [via a single configuration file](https://docs.platform.sh/configuration/app-containers.html).
+* [`pom.xml`](pom.xml) A Project Object Model or POM is the fundamental unit of work in Maven. It is an XML file that contains information about the project and configuration details used by Maven to build the project. It contains default values for most projects.
```

Comment:
This tells me what a POM file is if I don't know Java, but it doesn't tell me what's modified in the file to know what the Platform.sh-specific bits are that I should look for.  That's the goal for this section.

Reply:
> A Project Object Model or POM is the fundamental unit of work in Maven. It is an XML file that contains information about the project and configuration details used by Maven to build the project. It contains default values for most projects. This POM file has both add the config-reader-java and maven plugin to run the project at the Platform.sh.

What do you think?

Reply:
Is there any change other than "we added the config reader"?  If not, then we can remove this bullet entirely and just modify the following one to read "has been added to `pom.xml`."  The audience knows what a `pom.xml` file is.